indico/migrations/versions/20231222_1408_b37cbc4bb129_link_events_to_reservation_occurrences.py

```diff
@@ -0,0 +1,54 @@
+"""Link events to reservation occurrences.
+
+Revision ID: b37cbc4bb129
+Revises: 8e08236a529f
+Create Date: 2023-12-22 14:08:45.669303
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = 'b37cbc4bb129'
+down_revision = '8e08236a529f'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.rename_table('reservation_links', 'reservation_occurrence_links', schema='roombooking')
+    op.add_column('reservation_occurrences', sa.Column('link_id', sa.Integer(), nullable=True), schema='roombooking')
```

Comment:
change it to `nullable=False` after populating the field. that way you'll also know if your `UPDATE` query is missing anything